x/crosschain/types/expected_keepers.go

```diff
@@ -44,10 +44,9 @@ type BankKeeper interface {
 	MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
 }
 
-type ZetaObserverKeeper interface {
-	SetObserverMapper(ctx sdk.Context, om *observertypes.ObserverMapper)
-	GetObserverMapper(ctx sdk.Context, chain *common.Chain) (val observertypes.ObserverMapper, found bool)
-	GetAllObserverMappers(ctx sdk.Context) (mappers []*observertypes.ObserverMapper)
+type ObserverKeeper interface {
+	SetObservers(ctx sdk.Context, om observertypes.ObserverSet)
```

Comment:
minor: if correct, to make it consistent with below, 
`SetObserverSet`?

Reply:
I did not use the name as it sounds a bit weird to  Set-Observer-Set , but happy to make the change if it sounds okay to everyone else.

Reply:
@kingpinXD I added the change, I will push it to the branch

Reply:
[4481ff1](https://github.com/zeta-chain/node/pull/1440/commits/4481ff1bca76ebf5af9859b0dfbe7dcbcc8cdca4)